### PR TITLE
move recommended exchanges server side

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
tested on sim
based on #389 - merge after that

### CODE REVIEW

File: packages/daimo-common/src/model.ts [hunk 1]
```diff
@@ -75,3 +75,10 @@ export const zUserOpHex = z.object({
 });
 
 export type UserOpHex = z.infer<typeof zUserOpHex>;
+
+export const zRecommendedExchange = z.object({
+  cta: z.string(),
+  url: z.string(),
```
Comment (dcposch) [inline packages/daimo-common/src/model.ts:81]: love the simplicity

File: apps/daimo-mobile/src/view/screen/receive/DepositScreen.tsx [hunk 2]
```diff
@@ -67,34 +66,25 @@ function DepositScreenInner({ account }: { account: Account }) {
...
-        onPress={openCBPay}
-      />
+      {account.recommendedExchanges.map((rec, idx) => (
+        <View key={idx}>
+          <ButtonMed
+            type="primary"
+            title={rec.cta}
+            onPress={() => openRecommendedExchange(rec.url)}
+          />
+          {idx < account.recommendedExchanges.length - 1 && <Spacer h={16} />}
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/receive/DepositScreen.tsx:85]: might make only the first one primary and rest `subtle`

**Review by dcposch — APPROVED** [review]
LGreatTM

File: apps/daimo-mobile/src/view/screen/receive/DepositScreen.tsx (hunk 2 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/screen/receive/DepositScreen.tsx:85]: this is current:

**Pull request merged**